front/src/store/tv/__tests__/reducer.test.ts

```diff
@@ -0,0 +1,65 @@
+import { tvReducer } from '../reducers';
+import {
+  REQUEST_BASIC_SHOW_INFO_SUCCEEDED,
+  SAVE_EPISODE_DATA,
+  SAVE_SEARCH_QUERY,
+  SET_CALENDAR_EPISODES,
+} from '../actions';
+
+describe('tv reducer', () => {
+  const commonState = {
+    savedQueries: [{ query: '1' }],
+    episodeData: {},
+    basicShowInfo: {},
+    calendarEpisodesForDisplay: [],
+  };
+
+  it('saves query', () => {
+    const action = { type: SAVE_SEARCH_QUERY, payload: { query: '1' } };
+    expect(tvReducer(undefined, action)).toEqual(commonState);
+  });
+
+  it('does not add same query', () => {
+    const initialState = {
+      savedQueries: [{ query: '1' }] as any,
+      episodeData: {},
+      basicShowInfo: {},
+      calendarEpisodesForDisplay: [],
+    };
+    const action = { type: SAVE_SEARCH_QUERY, payload: { query: '1' } };
+    expect(tvReducer(initialState, action)).toEqual(commonState);
+  });
+
+  it('saves episode data', () => {
+    const expectedState = {
+      savedQueries: [],
+      episodeData: { episode: 'test' },
+      basicShowInfo: {},
+      calendarEpisodesForDisplay: [],
+    };
+    const action = { type: SAVE_EPISODE_DATA, payload: { episode: 'test' } };
+    expect(tvReducer(undefined, action)).toEqual(expectedState);
+  });
+
+  it('returns show info', () => {
+    const expectedState = {
+      savedQueries: [],
+      episodeData: {},
+      basicShowInfo: { 0: { episode: 'test' } },
+      calendarEpisodesForDisplay: [],
+    };
+    const action = { type: REQUEST_BASIC_SHOW_INFO_SUCCEEDED, payload: [{ episode: 'test' }] };
+    expect(tvReducer(undefined, action)).toEqual(expectedState);
+  });
+
+  it('returns calendar episode for display', () => {
```

Comment:
we should be consistent with the naming of the tests - so the last two say "returns" and the first few same "saves". it might be better for all the test names to follow a pattern

Reply:
Okay I will look for a more consistent pattern.